### PR TITLE
Feature/fix vswhere output

### LINKED ISSUE #3028: Cannot build using Conan and Visual Studio 2017 in Japanese Environment
Tests are with conan 1.4.4, Python 2.7.10, Windows 7

When trying to build a package in the local cache
 (conan install MyPackage/1.0.0@user/channel --build=missing)
I get the following error

        vcvars = tools.vcvars_command(self.settings)
        ValueError: Invalid \escape: line 12 column 54 (char 597)

This occurred after installing Visual Studio 2017.

The environment is Japanese Windows 7, but set to English environment. This sets the UI to English but many system strings still appear in Japanese.

Tracing through conan source shows the problem to be in the line
    return json.loads(vswhere_out)
of vswhere in win.py.

The output of vswhere.exe for VS2017 contains the line
 "description": "生産性向上と、さまざまな規模のチーム間の調整のための Microsoft DevOps ソリューション",
(This is line 12 of the vswhere.exe output)

Adding the code 

    output = '\n'.join(line for line in output.split('\n') if 'description' not in line)

to strip out this line from the vshwere.exe output before the line

    vswhere_out = decode_text(output).strip()

in vswhere in win.py allows the build to proceed normally.

So it seems the Japanese message in the json from vswhere is causing the loading of the json to fail.

When running vshwere.exe directly, the output is in utf-8 but this bug
 https://github.com/Microsoft/vswhere/issues/146
suggests that sometimes the output is not in utf-8.

### PR DESCRIPTION
**Pull request by lasote**
- [x] Refer to the issue that supports this Pull Request. Closes #3028

Seems like offending chars are in "description" field, we can strip that part

https://github.com/conan-io/docs/pull/685

### CODE REVIEW

File: conans/client/tools/win.py [hunk 1]
```diff
@@ -186,6 +186,8 @@ def vswhere(all_=False, prerelease=False, products=None, requires=None, version=
 
     try:
         output = subprocess.check_output(arguments)
+        output = "\n".join([line for line in output.splitlines()
+                            if not decode_text(line).strip().startswith('"description"')])
```
[the remark this answers is not in the document]
Comment (lasote) [reply conans/client/tools/win.py]: Yes, in py3 you cannot run startswith without decoding it. (bytes from output)

**Comment by lasote** [comment]
Interesting approvals due it is totally broken 🙄 🤣

**Comment by lasote** [comment]
Pending if the issue reported can try it: https://github.com/conan-io/conan/issues/3028

File: conans/client/tools/win.py [hunk 2]
```diff
@@ -5,6 +5,8 @@
 import deprecation
 
 import subprocess
+import six
```
Comment (memsharded) [inline conans/client/tools/win.py]: Finally six not used?

**Comment by lasote** [comment]
Confirmed by the reporter that it works

**Pull request merged**